cpp/src/bindings.cpp

```diff
@@ -114,6 +114,8 @@ std::shared_ptr<load_flow_parameters> initLoadFlowParameters() {
         parameters->countries_to_balance = pypowsybl::copyVectorStringToCharPtrPtr(configCountries);
         parameters->countries_to_balance_count = configCountries.size();
         parameters->connected_component_mode = config_params->connected_component_mode;
+        parameters->specific_parameters_keys_count = config_params->specific_parameters_keys_count;
```

Comment:
Naming: see in the issue, I think "provider_parameters" will be a better name everywhere than "specific_parameters" because we don't know with respect to what they are specific.